libopenfpga/libpinconstrain/src/pin_loc/pin_location.h

```diff
@@ -0,0 +1,47 @@
+#ifndef PIN_LOCATION
```

Comment:
The convention is ``PIN_LOCATION_H``

See example

https://github.com/lnis-uofu/OpenFPGA/blob/9dca626a3d7161c087d48eb8da5f7fad891b655b/libopenfpga/libopenfpgashell/src/command.h#L1-L2

Reply:
Fixed